crates/server/src/mcp/task_server.rs

```diff
@@ -247,7 +259,62 @@ impl TaskServer {
             client: reqwest::Client::new(),
             base_url: base_url.to_string(),
             tool_router: Self::tool_router(),
+            vk_context: None,
```

Comment:
```suggestion
            context: None,
```

super nit, but it's implied that this is relevant to vk!